functions/data_transformations.R

```diff
@@ -0,0 +1,64 @@
+install.packages("jsonlite")
```

Comment:
Don't add this in your files. We have all the necessary libraries loaded already :)